app/src/main/java/com/cpr3663/cpr_scouting_app/Match.java

```diff
@@ -213,7 +213,7 @@ else if ((rotation_degrees >= 180) && !currentOrientation.equals(ORIENTATION_LAN
         // Map the text box variable to the actual text box
         text_Time = matchBinding.textTime;
         // Initialize the match timer textbox settings
-        text_Time.setText(getString(R.string.timer_label) + TIMER_AUTO_LENGTH);
+        text_Time.setText(getString(R.string.timer_label) + " " + TIMER_AUTO_LENGTH);
```

Comment:
The timer_label already has a space.  this isn't needed, please remove.

Reply:
Remember how you needed to have the constant “       “ because it automatically trims off all of the spaces. Well it trims off the space.

Reply:
It trimmed what I wrote I wrote seven spaces it just doesn’t look like it.